.github/workflows/check-grafana-compatibility.yml

```diff
@@ -17,41 +17,37 @@ jobs:
     permissions:
       contents: read
     steps:
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      with:
-        persist-credentials: false
-        path: './grafana-plugin-sdk-go'
-    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-      with:
-        persist-credentials: false
-        repository: 'grafana/grafana'
-        path: './grafana'
-    - uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # 5.3.0
-      with:
-        cache: false
-        go-version: '~1.22'
-        check-latest: true
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+          path: "./grafana-plugin-sdk-go"
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
+          repository: "grafana/grafana"
+          path: "./grafana"
+      - uses: actions/setup-go@f111f3307d8850f501ac008e886eec1fd1932a34 # 5.3.0
+        with:
+          cache: false
+          go-version: "~1.22"
+          check-latest: true
 
-    - name: Check if branch exists in Grafana
-      working-directory: './grafana'
-      env:
-        BRANCH: ${{ github.head_ref }}
-      run: |
-        if git ls-remote --heads --quiet --exit-code origin "$BRANCH"; then
-          echo "Found branch $BRANCH in Grafana"
-          git fetch origin "$BRANCH" && git checkout "$BRANCH"
-        else
-          echo "Branch $BRANCH not found in Grafana"
-        fi
+      - name: Check if branch exists in Grafana
+        working-directory: "./grafana"
+        env:
+          BRANCH: ${{ github.head_ref }}
+        run: |
+          if git ls-remote --heads --quiet --exit-code origin "$BRANCH"; then
+            echo "Found branch $BRANCH in Grafana"
+            git fetch origin "$BRANCH" && git checkout "$BRANCH"
+          else
+            echo "Branch $BRANCH not found in Grafana"
+          fi
 
-    - name: Link sdk
-      working-directory: './grafana'
-      run: go mod edit -replace github.com/grafana/grafana-plugin-sdk-go=../grafana-plugin-sdk-go
+      - name: Link sdk
+        working-directory: "./grafana"
+        run: go mod edit -replace github.com/grafana/grafana-plugin-sdk-go=../grafana-plugin-sdk-go
 
-    - name: Automatic updates
-      working-directory: './grafana'
-      run: go mod tidy
```

Comment:
FTR: This step is not needed because the build-go make target runs `go mod tidy` nowadays